### PR TITLE
Feature/humbug

### PR DESCRIPTION
**Pull request by edwinkortman**
Solves #19.
You can use humbug by running `vendor/bin/humbug run`. 
Snippet of the output:

> **25 mutations were generated:**
        24 mutants were killed
        0 mutants were not covered by tests
        1 covered mutants were not detected
        0 fatal errors were encountered
        0 time outs were encountered

> **Metrics:**
        Mutation Score Indicator (MSI): 96%
        Mutation Code Coverage: 100%
        Covered Code MSI: 96%

### CODE REVIEW

**Comment by jaspernbrouwer** [comment]
Why do we output logs in json _and_ txt?

**Comment by jaspernbrouwer** [comment]
And let's put it in Travis!

**Comment by edwinkortman** [comment]
Output logs are by default in json and text so a developer can pick it's own favourite. Humbug favours using your local settings so if you prefer having one file in your editor you can override it. :-)

Added to `travis.yml` as done accordingly!

File: docs/lessons_learned.md [hunk 1]
```diff
@@ -46,3 +46,10 @@ access to routes from within javascript. I followed the manual
 
 ---
 
+**name:** Edwin Kortman
+
+**I learned about:** Humbug Mutation Testing! I was aware of the code variety that could exist,
```
Comment (ramondelafuente) [inline docs/lessons_learned.md:51]: <3

**Pull request merged**